src/core/p5.Element.js

```diff
@@ -28,13 +28,14 @@ p5.Element = function(elt, pInst) {
    * @example
    * <div>
    * <code>
-   * createCanvas(300, 500);
-   * background(0, 0, 0, 0);
-   * var input = createInput();
-   * input.position(20, 225);
-   * var inputElem = new p5.Element(input.elt);
-   * inputElem.style('width:450px;');
-   * inputElem.value('some string');
+   * function setup() {
+   *   var c = createCanvas(50, 50);
+   *   c.elt.style.border='5px solid red';
```

Comment:
the linter is complaining here, I'm not totally sure why your grunt call didn't flag it
but can you add spaces on either side of the `=` here